module/x/peggy/types/batch.go

```diff
@@ -32,6 +32,8 @@ type OutgoingTxBatch struct {
 	TotalFee           ERC20Token           `json:"total_fee"`
 	BridgedDenominator BridgedDenominator   `json:"bridged_denominator"`
 	BatchStatus        BatchStatus          `json:"batch_status"`
+	Valset             Valset               `json:"valset"`
```

Comment:
Q: Do you need the full `Valset` here? Only `b.Valset.GetCheckpoint` is used so far.

Reply:
you need the full valset to create the checkpoint, in theory we could only store the checkpoint but I don't see a downside to carrying all the data with us.

Reply:
There's a question about whether you keep the valset in the type or if you pass it in when calling GetDigest. It probably won't make a huge difference, but I thought it might be nicer to have GetDigest be a pretty stateless method